postman/AskDarcel%20API.postman_collection.json

```diff
@@ -593,6 +593,174 @@
 			},
 			"response": []
 		},
+		{
+			"name": "Get services by category using algolia endpoint",
```

Comment:
nit: "Get services by multiple categories using algolia endpoint"

Reply:
updated this.